core/controllers/base.py

```diff
@@ -341,9 +340,14 @@ def vaidate_and_normalize_args(self):
         payload_arg_keys = []
         request_arg_keys = []
         for arg in self.request.arguments():
-            if arg in self.ARGS_WHICH_DO_NOT_NEED_SCHEMA_VALIDATION:
+            if arg == 'csrf_token':
```

Comment:
The csrf_token is ignored because -- validate_normalize_args() method is called from dispatch method and in the dispatch method we already validate whether it is correct or not, so need to validate twice.

Reply:
Please add this as a comment.

Reply:
Done, ptal